app/panel/components/BuildingBlocks/StatsGraph.jsx

```diff
@@ -161,6 +161,15 @@ class StatsGraph extends React.Component {
 				.attrTween('stroke-dasharray', interpolator);
 		}
 
+		function getThemeColor(theme) {
+			switch (theme) {
+				default:
+					return '#124559';
+				case 'palm-theme':
+					return '#172a0b';
+			}
```

Comment:
Order of operations matter here. Default cases must be the last case otherwise it will always be skipped. In this case this switch statement will always result in '#172a0b'.

Reply:
Here is an example: https://www.w3schools.com/code/tryit.asp?filename=GD0BISHI16P6
(Don't judge me for using w3schools code for it...)